CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing
+
+First of all, thank you for your interest in contributing.
+
+## Bug reports / feature requests
+
+If you experience any bugs or have feature requests, please [file an issue].
+
+[file an issue]: https://github.com/slide-rs/specs/issues/new
+
+## Getting started
+
+If you want to contribute code, please read the following sections.
+
+There are couple of recommended step before you start working on a ticket:
+
+1. If you haven't already, please read [the Specs book](https://slide-rs.github.io/specs/)
+2. Please make sure you read our [Code of Conduct](CODE_OF_CONDUCT.md)
+3. Refer to the [architecture section](#architecture) below to gain some overview
+4. Please continue with the next section (Creating Pull Requests)
+
+## Creating Pull Requests
```

Comment:
Maybe better to move this under #git

Reply:
Hmm, "# Git" was about Git basics, so I think this section is actually fine here.